test/queries.js

```diff
@@ -166,3 +281,9 @@ graphql(BigchainDBSchema, queryVoteList).then(result => {
 graphql(BigchainDBSchema, querySearch).then(result => {
     console.log(JSON.stringify(result, null, 2))
 })
+graphql(BigchainDBSchema, mutationTransaction).then(result => {
```

Comment:
you think we can do it without the python? this would make it easier for our users... (only need to npm install)